content/api/commands/intercept.md

```diff
@@ -22,136 +22,64 @@ Unlike [cy.route()](/api/commands/route), `cy.intercept()`:
 - does not require calling [cy.server()](/api/commands/server) before use - in fact, `cy.server()` does not influence `cy.intercept()` at all.
 - does not have method set to `GET` by default, but intercepts `*` methods.
 
-## Usage
+## Syntax
 
-```ts
-cy.intercept(url, routeHandler?)
-cy.intercept(method, url, routeHandler?)
-cy.intercept(routeMatcher, routeHandler?)
+### Route-matching Only (No Stubbing or Intercepting)
+```js
+cy.intercept(url)
+cy.intercept(method, url)
+cy.intercept(routeMatcher)
+cy.intercept(url, routeMatcher)
+cy.intercept(method, url, routeMatcher)
+```
+
+### Route-matching With Stubbing and Intercepting
+```js
+cy.intercept(url, routeHandler)
+cy.intercept(method, url, routeHandler)
+cy.intercept(routeMatcher, routeHandler)
 cy.intercept(url, routeMatcher, routeHandler)
+cy.intercept(method, url, routeMatcher, routeHandler)
 ```
 
 **Note:** all intercepts are automatically cleared before every test.
 
+### Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```js
+cy.intercept('/users/**')
+```
 ### Arguments
 
 #### **<Icon name="angle-right"></Icon> url** **_(`string | RegExp`)_**
 
 Specify the URL to match. See the examples for [Matching URL](#Matching-URL) to see how URLs are matched.
```

Comment:
Implied standards for an argument in an Arguments section:
* not in a header, but bold and camelCase
* prefixed with an angle-right icon
* types are italicized and PascalCase
* types are wrapped in parens and separated by commas